src/cli/console.ts

```diff
@@ -111,6 +111,98 @@ export class Console {
 
     return text;
   }
+
+  // Enhanced styling methods for better DX with consistent colors
+  title(message: string) {
+    const text = chalk.bold.white(message);
+    if (!this.options.quiet) {
+      global.console.log(text);
+    }
+    return text;
+  }
```

Comment:
question: Isn't it the same as `header`? 

thought: This class is becoming a bit too much, I would have no clue what to use if I was to write something from scratch. We should reorganize it at some point. Since most of these are used in `diff` (I think), maybe diff could have its own extended console and we would use the "basic" one in all the other places